setup.cfg

```diff
@@ -1,8 +1,15 @@
 [mypy]
+mypy_path = stubs
+no_implicit_optional = True
 warn_unused_ignores = True
 strict_optional = True
 check_untyped_defs = True
 ignore_missing_imports = True
+disallow_subclassing_any = True
```

Comment:
Wary about adding quite so many additional settings without context

Reply:
These settings essentially ensure that new code that is added has types on it, so I don't have to go through and type things again.